peachjam/templates/peachjam/_document_table_form.html

```diff
@@ -0,0 +1,132 @@
+{% load i18n %}
+<form method="get"
+      id="filter-form"
+      data-component="DocumentFilterForm"
+      hx-include="#filter-form"
+      hx-indicator=".htmx-progress"
+      hx-target="#filter-form"
+      hx-swap="outerHTML">
+  <div class="d-block d-lg-none my-2">
+    <a class="btn btn-primary"
+       href="#document-list-filters-offcanvas"
+       data-bs-toggle="offcanvas"
+       aria-controls="document-list-filters-offcanvas">
+      {% trans 'Filters' %}
+    </a>
+  </div>
+  <div class="offcanvas offcanvas-start"
+       tabindex="-1"
+       id="document-list-filters-offcanvas"
+       aria-labelledby="document-list-filters-offcanvas-title">
+    <div class="offcanvas-header">
+      <button type="button"
+              class="btn-close ms-auto"
+              data-bs-dismiss="offcanvas"
+              aria-label="{% trans "Close" %}"></button>
+    </div>
+    <div class="offcanvas-body"></div>
+  </div>
+  <div class="row">
+    <div class="col-lg-3 d-none d-lg-block document-list-facets-wrapper">
+      <div>
+        {% block desktop-taxonomy-list %}{% endblock %}
+      </div>
+      <ul class="list-group document-list-facets pocketlaw-hidden">
+        <li class="position-relative list-group-item bg-light d-flex justify-content-between align-items-center">
+          <strong id="document-list-filters-offcanvas-title">{% trans 'Filters' %}</strong>
+        </li>
+        {% for facet_name, facet in facet_data.items %}
+          {% if facet.options %}
+            <li class="list-group-item">
+              <div class="d-flex justify-content-between mb-2">
+                <strong>{{ facet.label }}</strong>
+                <div class="d-flex align-items-center">
+                  <a href="{{ request.path }}"
```

Comment:
Can we have this only if `facet.values` isn't empty?

Reply:
same applies for the other "Clear" options

Reply:
done